### PR TITLE
Drop Julia 0.6 and 0.7

### PR DESCRIPTION
**Pull request by omus**
Replaces #5

### CODE REVIEW

**Comment by omus** [comment]
Looks like I managed to mess up the dependencies at the last second

File: Project.toml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+name = "Syslogs"
+uuid = "cea106d9-e007-5e6c-ad93-58fe2094e9c4"
+authors = ["Rory Finnegan <rory.finnegan@gmail.com>"]
```
Comment (nicoleepp) [inline Project.toml]: Should be "Invenia Technical Computing Corporation"

File: docs/make.jl [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
-    sitename="Syslogs.jl",
-    authors="Invenia Technical Computing Corporation",
-    assets=[
+    repo = "https://github.com/invenia/Syslogs.jl/blob/{commit}{path}#L{line}",
+    sitename = "Syslogs.jl",
+    authors = "Invenia Technical Computing Corporation",
+    assets = [
         "assets/invenia.css",
-     ],
+    ],
```
Comment (fchorney) [inline docs/make.jl:14]: I was going to comment that I thought the standard was no spaces around the `=` sign, but looking at other projects, it seems like we do both in the same file in various projects.

File: src/Syslogs.jl [hunk 3]
```diff
@@ -120,7 +118,7 @@ Syslog(host::IPAddr, facility::Symbol=:user; tcp::Bool=false) =
     Syslog(host, tcp ? TCP_PORT : UDP_PORT, facility; tcp=tcp)
 
 Syslog(facility::Symbol=:user) =
-    Syslog(Nullable{Tuple{IPAddr, Int}}(), facility, Nullable{Base.LibuvStream}())
+    Syslog(nothing, facility, nothing)
```
Comment (nicoleepp) [inline src/Syslogs.jl]: Can fit on the previous line now

File: docs/make.jl (hunk 2 above)
Comment (omus) [reply docs/make.jl:14]: Ya, I'm not sure why this file seems to be special

File: Project.toml (hunk 1 above)
Comment (omus) [reply Project.toml]: Just to check: have we documented this somewhere?

File: REQUIRE [hunk 4]
```diff
@@ -1,3 +1 @@
-julia 0.6
-Compat 0.65.0
-Nullables 0.0.3
+julia 1.0
```
Comment (nicoleepp) [inline REQUIRE:1]: Can we get rid of the REQUIRE file completely now? I don't think anything uses it anymore

Comment (omus) [reply REQUIRE:1]: Yes, I think you are correct in that this file can now be deleted. I haven't gone through the new Registrator process yet so I was being cautious.

**Review by nicoleepp — APPROVED** [review]

**Pull request merged**